.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -70,10 +70,29 @@ jobs:
         -DAVIF_CODEC_DAV1D=ON
         -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON
         -DAVIF_BUILD_TESTS=ON -DAVIF_ENABLE_GTEST=ON -DAVIF_LOCAL_GTEST=ON
-        -DAVIF_BUILD_GDK_PIXBUF=ON
+        -DAVIF_BUILD_GDK_PIXBUF=ON -DCMAKE_INSTALL_PREFIX=./install/
```

Comment:
Nit: `CMAKE_INSTALL_PREFIX` does not need to end with '/', right?

Reply:
Done